distrho/src/DistrhoUI.cpp

```diff
@@ -40,7 +40,7 @@ Window*     d_lastUiWindow     = nullptr;
 // -----------------------------------------------------------------------------------------------------------
 
 #if DISTRHO_PLUGIN_HAS_EXTERNAL_UI
-UI* createUiWrapper(void* const dspPtr, const uintptr_t winId, const double scaleFactor, const char* const bundlePath)
+UI* createUiWrapper(void* dspPtr, uintptr_t winId, double scaleFactor, const char* bundlePath)
```

Comment:
const attributes were removed, likely a mistake

Reply:
It is not a mistake: the declarations and the definitions of these fuctions will get different mangling, when the arguments differ in constness.

Reply:
but they dont. the const is related to the variable access, not its type.
It is perfectly valid to do something like this:
```
// header stuff
int somefunc(int x);
// implementation
int somefunc(const int x) { return x; }
```

Reply:
I purposefully do not use `const` on the headers to make it simpler/nicer to look at.
But implementation should use const whenever possible.

Reply:
I assure you 100%, that on MSVC compiler, contrary to GCC, the constness of parameters affects the mangling.
This mismatches definition and declaration, and this fails the link.

Reply:
I trust you. Just very odd then, because the same "issue" is used through-out the code in DPF.
It is complaining about these in particular for some reason..
Please change the declaration on DistrhoUIInternal.hpp to include the const attribute, instead of removing them here.
Thanks!